### PR TITLE
added score bubble for assignments on student dashboard

### PR DESCRIPTION
**Pull request by alvinwan**
assignment score bubble

screen shot 2015-02-04 at 11 47 25 am

### CODE REVIEW

**Comment by alvinwan** [comment]
bubble blends in to assignment color

screen shot 2015-02-04 at 11 49 49 am

**Comment by moowiz** [comment]
That looks kinda small and hard to read from the screenshot I saw... But I
could be wrong. Looking at it on my iPhone.

**Comment by alvinwan** [comment]
@moowiz in terms of color contrast? or just size? I can blow it up and/or shift it to the right of the assignment.

File: server/static/partials/student/assignment.dash.html [hunk 1]
```diff
@@ -2,11 +2,12 @@
 
   <div class="assign large active" id="1" ng-repeat="assign in assignments">
     <div class="assign-main">
+      <div class="score">{{ ::assign.assignment.score }}<span class="score-total">/{{::assign.assignment.points}}</span><span class="score-pts">PTS</span></div>
       <img class="assign-bg" src="/static/student/images/bg-clock.png">
        <h2 class="assign-title"> {{ ::assign.assignment.display_name }}</h2>
       <div class="assign-deck">
         <p><img src="/static/student/images/icon-clock.png">due {{ ::assign.assignment.due_date | amCalendar }}</p>
-        <p><span class="icon-question">?</span> {{::assign.assignment.points}} points</p>
+        <p><span class="icon-question">?</span> --/{{::assign.assignment.points}} points</p>
```
Comment (Sumukh) [inline server/static/partials/student/assignment.dash.html]: Can you remove this line too. We'll just rely on the bubble for the grade and if it's ungraded the bubble will say --/25

**Comment by Sumukh** [comment]
feel free to make the grading part bigger if you want to. I think @moowiz was referring to size. 

I'll merge and deploy once you decide if you want to change the size and delete the line I commented on.

**Comment by alvinwan** [comment]
Ohh okay, gotcha. Making the changes right now.

**Comment by alvinwan** [comment]
New, larger bubble for scores

screen shot 2015-02-04 at 8 43 41 pm

**Comment by moowiz** [comment]
Yeah I was talking about size. That looks much better.

**Pull request merged**